packages/expect/src/matchers.ts

```diff
@@ -607,7 +611,7 @@ const matchers: MatchersObject = {
           return (
             matcherHint('.toHaveProperty', 'object', 'path', {
```

Comment:
`matcherHint` should return `string`, shouldn't have to cast here

Reply:
Not sure what you mean here. Was typing the options as a `MatcherHintOptions`.

Reply:
I misread the cast! Why do you need to cast on lin 616? Shouldn't be needed